src/components/ChartModal/BiAxialChartConfig/index.tsx

```diff
@@ -0,0 +1,386 @@
+                const option = options.find(item => item.key === bar.optionName);
+
+                if (!option) {
+                    return undefined;
+                }
+
+                return {
+                    title: option.title,
+                    valueSelector: option.valueSelector,
+                    color: bar.color,
+                    dependency: option.dependency,
+                    type: bar.type,
+                };
+            }).filter(isDefined);
+
+            if (properBars.length <= 1) {
+                setError('Two bars should be set.');
+                return;
+            }
+
+            if (isFalsyString(limitValue)) {
+                setError('Limit value on data points is required.');
+                return;
+            }
+
+            const limit = +limitValue;
+            if (limit <= 0) {
+                setError('Limit value must be greater than zero.');
+                return;
+            }
+
+            if (isFalsyString(autoOrderField)) {
+                setError('Field to order data points is required.');
+                return;
+            }
+
+            const orderOption = options.find(item => item.key === autoOrderField);
+            if (!orderOption) {
+                setError('Field to order data points is required.');
+                return;
+            }
+
+            const nonUniqueDependencies = properBars.map(item => item.dependency);
+            nonUniqueDependencies.push(orderOption.dependency);
+
+            const dependencies = unique(
+                nonUniqueDependencies.filter(isDefined),
+                item => item,
+            );
+
+            const settings: BiAxialChartSettings<T> = {
+                id: chartId,
+                type: 'bi-axial-chart',
+                title,
+
+                keySelector: primaryKeySelector,
+
+                limit: {
+                    count: limit,
+                    method: order === 'asc' ? 'min' : 'max',
+                    valueSelector: orderOption.valueSelector,
+                },
+
+                chartData: properBars,
+
+                dependencies,
+            };
+
+            onSave(settings);
+        },
+        [
+            onSave, biAxialData, title, options, limitValue,
+            order, autoOrderField, primaryKeySelector,
+        ],
+    );
+
+    const onToggleChartType = useCallback(() => {
+        const [firstData, secondData] = biAxialData;
+        if (firstData.type === 'bar') {
+            const tmpFD: BiAxialData = { ...firstData, type: 'line' };
+            const tmpSD: BiAxialData = { ...secondData, type: 'bar' };
+            setBiAxialData([tmpFD, tmpSD]);
+        }
+        if (firstData.type === 'line') {
+            const tmpFD: BiAxialData = { ...firstData, type: 'bar' };
+            const tmpSD: BiAxialData = { ...secondData, type: 'line' };
+            setBiAxialData([tmpFD, tmpSD]);
+        }
```

Comment:
You don't really need to do this but have same logic as inverting the type using map on that view page.